src/options.h

```diff
@@ -34,6 +34,8 @@ typedef struct {
 	int temp_set;
 	/* Whether to fade between large skips in color temperature. */
 	int use_fade;
+	/* The length of the fade duration in seconds */
```

Comment:
This should probably link to the spec "fade/duration" and not repeat the description?

Reply:
Yes, thanks for the hint!

Reply:
Fixed.